external-docs/docs/guides/configuration/qube-config/columns/attributes/attribute-values.md

```diff
@@ -0,0 +1,185 @@
+# Attribute values configuration
+
+This page discusses what attribute values are, where they should be used, and how they can be defined.
+
+See the [Attributes page](./index.md) for general information about attribute columns, including when to use one,
+and a discussion of the difference between [Resource](./attribute-resources.md) attributes and
+[Literal](./attribute-literals.md) attributes.
+
+> For a detailed look at attribute values configuration options, see the [Reference table](#reference) at the
+bottom of this page.
+
+## What are attribute values?
+
+Attribute values are the cell values in the attribute column of your CSV file. csvcubed will automatically generate
+these values from the cell values if you specify the `values` field as `true`. However, it is also possible to
+explicitly configure attribute values, as described below.
```

Comment:
Can we have an example table up here containing the attribute values we refer to later in the docs?

Reply:
Added to the "Basic configuration" section